scripts/provisioning/s3_prov_config.yaml

```diff
@@ -49,8 +49,8 @@ INIT:
 # Inheritance hierarchy ends here.
 
 TEST:
-  CONFSTORE_LDAPADMIN_USER_KEY: "cortx>software>openldap>sgiam>user"
-  CONFSTORE_LDAPADMIN_PASSWD_KEY: "cortx>software>openldap>sgiam>secret"
+  CONFSTORE_LDAPADMIN_USER_KEY: "openldapiamuser"
+  CONFSTORE_LDAPADMIN_PASSWD_KEY: "openldapiamsecret"
```

Comment:
Why not put hierarchy :
cortx>software>openldap>sgiam>user
cortx>software>openldap>sgiam>password

Reply:
The only concern I have is how do we maintain the isolation?
Thats why I recommended a flat hirarchy @nitin-seagate and I think we should pursue the same if there is no strong reason for adding it to cortx.